schema/sql/api.sql

```diff
@@ -125,6 +126,23 @@ comment on column v1.ssh_public_key.uid is
 alter view v1."ssh_public_key" owner to api;
 grant select on table v1."ssh_public_key" to "api-anon";
 
+create view v1.gpg_public_key as
+  select
+      encode(fingerprint, 'hex') as fingerprint,
+      encode(key, 'ascii') as key,
+      uid
+  from public.gpg_public_key;
+comment on view v1.gpg_public_key is
+    $$PGP public keys for users$$;
+comment on column v1.gpg_public_key.fingerprint is
+    $$Fingerprint for PGP public key$$;
+comment on column v1.gpg_public_key.key is
+    $$ASCII armoured PGP key$$;
```

Comment:
I think this means that this was /intended/ to be an ASCII armored public key?

Reply:
As per discussion on IRC, this should instead be a bytea field that is the "raw PGP key". It can|should be formatted as ASCII armored using `pgcrypto.armor()`.